CMakeLists.txt

```
@@ -20,6 +20,9 @@ option(C2A_USE_ALL_CORE_APPS        "Use C2A-core all Applications" ON)
 option(C2A_USE_ALL_CORE_TEST_APPS   "Use C2A-core all Test Applications" ON)
 option(C2A_USE_ALL_CORE_LIB         "Use C2A-core all library" ON)
 
+## C2A build target board option （標準実装などがこのオプションによってスイッチされる．デフォルトでは sub OBC を想定）
+option(C2A_ENABLE_MOBC_FEATURES     "Enable some features for MOBC" OFF)
```

enable は動作なので，要らないと思う

```suggestion
option(C2A_MOBC_FEATURES     "Enable some features for MOBC" OFF)
```

---

`C2A_USE_ALL_CORE_APPS` だしそうか．
ドキュメントとあわせて直しました．

---

enable flag を enable したり disable したりすることになっちゃいますからね